### PR TITLE
[MIG] 14.0 agreement_sale

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

File: agreement_sale/models/sale.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    agreement_id = fields.Many2one(
+        comodel_name="agreement",
+        string="Agreement",
+        ondelete="restrict",
+        track_visibility="onchange",
```
Comment (patrickrwilson) [inline agreement_sale/models/sale.py]: v13 changed the track_visibility attribute and it should now be tracking=True.

File: agreement_sale/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        readonly=True,
+        copy=False,
+        states={"draft": [("readonly", False)], "sent": [("readonly", False)]},
+    )
+
+    agreement_type_id = fields.Many2one(
+        comodel_name="agreement.type",
+        string="Agreement Type",
+        ondelete="restrict",
+        track_visibility="onchange",
```
Comment (patrickrwilson) [inline agreement_sale/models/sale.py]: v13 changed the track_visibility attribute and it should now be tracking=True

File: agreement_sale/views/agreement.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
```
Comment (patrickrwilson) [inline agreement_sale/views/agreement.xml]: This line is no longer needed.

File: agreement_sale/views/res_config_settings.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline agreement_sale/views/res_config_settings.xml]: This line is no longer needed.

File: agreement_sale/views/sale_order.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline agreement_sale/views/sale_order.xml]: This line is no longer needed.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Just a few requested changes.
- no longer needed xml lines
- tracking_visibility change

**Comment by PierrickBrun** [comment]
@patrickrwilson Done and rebased.

I will need someone to approve the workflow but idk who to ping

**Review by patrickrwilson — APPROVED** [review]
The only thing left that you should squash the [UPD] and translation commits to clean up the commit history and remove all the automated commits. 

Other than that it looks good.

**Comment by patrickrwilson** [comment]
@max3903 @dreispt @osi-scampbell @ReedHayashikawa can you help review please?

**Review by rconjour — APPROVED** [review]
LGTM

**Review by smangukiya — APPROVED** [review]
LGTM

**Pull request merged**